ssm_jax/linear_gaussian_ssm/demos/blocked_gibbs_lgssm.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # (averaged over 50 consecutive iterations to smooth the curve)
</pre><p>50 -&gt; 10</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/ssm-jax/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>